### PR TITLE
[docs][yba][DOC-73] YBA trust store

### PR DESCRIPTION
**Pull request by ddhodge**
YBA trust store

@netlify /preview/yugabyte-platform/security/enable-encryption-in-transit/#use-self-signed-certificates

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 1]
```diff
@@ -632,3 +633,48 @@ In addition, as the `ssl_protocols` setting does not propagate to PostgreSQL, it
...
+
+1. Click **Upload**, select your certifcate (in .crt format) and click **Save CA Certificate**.
+
+### Rotate a certificate in your trust store
+
+To rotate a certificate in your YugabyteDB Anywhere Trust Store, do the following:
+
+1. Navigate to **Admin > CA Certificates**.
+
+1. Click thew **...** button for the certificate and choose **Update Certificate**.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:668]:
```suggestion
1. Click the **...** button for the certificate and choose **Update Certificate**.
```

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 2]
```diff
@@ -632,3 +633,48 @@ In addition, as the `ssl_protocols` setting does not propagate to PostgreSQL, it
...
+
+1. Click **Upload**, select your certifcate (in .crt format) and click **Save CA Certificate**.
+
+### Delete a certificate in your trust store
+
+To delete a certificate in your YugabyteDB Anywhere Trust Store, do the following:
+
+1. Navigate to **Admin > CA Certificates**.
+
+1. Click thew **...** button for the certificate and choose **Delete**, then click **Delete CA Certificate**.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:680]:
```suggestion
1. Click the **...** button for the certificate and choose **Delete**, then click **Delete CA Certificate**.
```

File: docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 3]
```diff
@@ -632,3 +632,50 @@ In addition, as the `ssl_protocols` setting does not propagate to PostgreSQL, it
...
+
+1. Click **Upload**, select your certifcate (in .crt format) and click **Save CA Certificate**.
+
+### Rotate a certificate in your trust store
+
+To rotate a certificate in your YugabyteDB Anywhere Trust Store, do the following:
+
+1. Navigate to **Admin > CA Certificates**.
+
+1. Click thew **...** button for the certificate and choose **Update Certificate**.
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md:669]:
```suggestion
1. Click the **...** button for the certificate and choose **Update Certificate**.
```

File: docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 4]
```diff
@@ -632,3 +632,50 @@ In addition, as the `ssl_protocols` setting does not propagate to PostgreSQL, it
...
+
+1. Click **Upload**, select your certifcate (in .crt format) and click **Save CA Certificate**.
+
+### Delete a certificate in your trust store
+
+To delete a certificate in your YugabyteDB Anywhere Trust Store, do the following:
+
+1. Navigate to **Admin > CA Certificates**.
+
+1. Click thew **...** button for the certificate and choose **Delete**, then click **Delete CA Certificate**.
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md:681]:
```suggestion
1. Click the **...** button for the certificate and choose **Delete**, then click **Delete CA Certificate**.
```

**Review by aishwarya24 — APPROVED** [review]
LGTM

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 5]
```diff
@@ -632,3 +633,48 @@ In addition, as the `ssl_protocols` setting does not propagate to PostgreSQL, it
 ```shell
 --ysql_pg_conf_csv="ssl_min_protocol_version=TLSv1.2"
 ```
+
+## Use self-signed and custom CA certificates
+
+YugabyteDB Anywhere uses TLS to protect data in transit when connecting to other services, including:
+
+- LDAP
+- OIDC
```
Comment (ankit-yb) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:642]: OIDC is not going in 2.18.2

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 6]
```diff
@@ -632,3 +633,48 @@ In addition, as the `ssl_protocols` setting does not propagate to PostgreSQL, it
...
+
+### Rotate a certificate in your trust store
+
+To rotate a certificate in your YugabyteDB Anywhere Trust Store, do the following:
+
+1. Navigate to **Admin > CA Certificates**.
+
+1. Click the **...** button for the certificate and choose **Update Certificate**.
+
+1. Enter a name for the certificate.
```
Comment (ankit-yb) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:670]: We cannot update the name (we can only change the certificate)

File: docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 7]
```diff
@@ -632,3 +632,50 @@ In addition, as the `ssl_protocols` setting does not propagate to PostgreSQL, it
 ```shell
 --ysql_pg_conf_csv="ssl_min_protocol_version=TLSv1.2"
 ```
+
+## Use self-signed and custom CA certificates
+
+YugabyteDB Anywhere uses TLS to protect data in transit when connecting to other services, including:
+
+- LDAP
+- OIDC
```
Comment (ankit-yb) [inline docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md:641]: OIDC is not going in 2.18.2

File: docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 8]
```diff
@@ -632,3 +632,50 @@ In addition, as the `ssl_protocols` setting does not propagate to PostgreSQL, it
...
+
+### Rotate a certificate in your trust store
+
+To rotate a certificate in your YugabyteDB Anywhere Trust Store, do the following:
+
+1. Navigate to **Admin > CA Certificates**.
+
+1. Click the **...** button for the certificate and choose **Update Certificate**.
+
+1. Enter a name for the certificate.
```
Comment (ankit-yb) [inline docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md:671]: We cannot update the name (we can only change the certificate)

**Review by ankit-yb — CHANGES_REQUESTED** [review]

**Pull request merged**